src/pages/add-products/components/product-finish/validation.ts

```diff
@@ -0,0 +1,47 @@
+import * as Yup from "yup";
+
+import { categoryTypeStyleValidation } from "../product-category-and-type/validation.ts";
+import { productDescriptionValidation } from "../product-description-form/validation.ts";
+import { productImagesValidation } from "../product-images/validation.ts";
+import { productVariantsValidation } from "../product-variants/validation.ts";
+
+const productDetailsValidationMessage = {
+	INVALID_FORMAT: "Price must be in the format 'min-max', e.g., '100-200'",
+	INVALID_RANGE: "Invalid price range. Minimum must be less than maximum",
+	REQUIRED_PRICE: "Price is required",
+};
```

Comment:
can we move it to i18 library, its will be difficult to translate if one more language will be added